tests/e2e/keycloak/01-connect.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: connect-keycloak-e2e
+spec:
+  backoffLimit: 20
+  template:
+    metadata:
+      labels:
+        e2e-test: keycloak-e2e
+    spec:
+      restartPolicy: Never
+      containers:
+        - name: connect
+          image: docker.io/appuio/oc:v4.14
+          imagePullPolicy: IfNotPresent
+          command:
+            - bash
+          args:
+            - -c
+            - echo "Testing health...\n" && sleep 30 && curl -ivv http://$KEYCLOAK_HOST/health
```

Comment:
I'm not sure if we really need this curl. I guess the deployment is already checking this same endpoint. As soon as the deployment is ready then it is reflected in the status field object thus making this job obsolete? @Kidswiss what do you think? I would rather do other checks instead of curl if we want to be sure the intance is available from outside the deployment management.

Reply:
that was advised by @Kidswiss

Reply:
@zugao There's currently no assert for the deployment status, so it's not really obsolete.

Also in the spirit of end-to-end test, this curl does more than the liveness probe of the pod:
* It uses the `KEYCLOAK_HOST`
* It connects from outside -> network policies

We can make it do some authenticated call in a future iteration as well.